ignite/contrib/handlers/tqdm_logger.py

```diff
@@ -49,7 +49,11 @@ def __call__(self, engine, logger, event_name):
         if logger.pbar is None:
             logger._reset(pbar_total=pbar_total)
 
-        desc = self.tag
+        state_dict = engine.state_dict()
+        max_epochs = state_dict["max_epochs"] if "max_epochs" in state_dict else None
```

Comment:
Can you use `engine.state.max_epochs` instead of `engine.state_dict()`. Here, `engine.state.max_epochs` can not be `None`.

Reply:
Ok